apps/web-giddh/src/app/settings/integration/payment/icici/account-create-edit/account-create-edit.component.html

```diff
@@ -46,7 +46,7 @@ <h3 class="modal-title">{{ activeBankAccount ? localeData?.payment?.edit_account
                             <ng-container>
                                 <a href="javascript:void(0)" class="list-item d-flex">
                                     <span><input type="checkbox"
-                                            [checked]="((selectAllRecords !== option.value && paymentAlerts?.includes(option.value)) || selectAllRecords === option.value && (paymentAlerts?.length === paymentAlertsUsersList?.length || paymentAlerts?.length === paymentAlertsUsersList?.length - 1))"
+                                            [checked]="(selectAllRecords !== option.value && paymentAlerts?.includes(option.value)) || isSelectAllChecked(option.value)"
```

Comment:
Avoid usage of methods in template: `isSelectAllChecked` this will be executed on every change detection even if result recalculation is not required